affiliated/index.html

```diff
@@ -255,6 +255,10 @@ <h1 id="becoming-an-affiliated-package">Becoming an Affiliated Package<a class="
 		consistent with basic principles of open source. (E.g.,
 		<a href="https://choosealicense.com/">an OSI-approved license</a>)</li>
 
+		<li>Include instructions to users on how to cite your package. (This is commonly done
+			with a CITATION.md file.) Also include citations to other relevant papers and software following
```

Comment:
```suggestion
			with a CITATION file.) Also include citations to other relevant papers and software following
```
The core package itself and all coordinated packages I have looked up are using either plain name or `CITATION.rst`;-) if we want to endorse .md (not actually sure what this implies for format or representation), we should at least, or most, put `CITATION or CITATION.md` at this point.